skietbaan_fe/src/components/NavbarMenuUser.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import { Navbar } from 'react-bootstrap';
+import '../scss/navbar.css';
+import 'font-awesome/css/font-awesome.min.css';
+import '../bootstrap/NavbarMenuStyle.css';
+
+class NavbarMenuUser extends Component {
+	constructor(props) {
+		super(props);
+
+		this.isActive = this.isActive.bind(this);
+		this.isHome = this.isHome.bind(this);
+		this.isScoreCapture = this.isScoreCapture.bind(this);
+		this.isProfile = this.isProfile.bind(this);
+		this.isDocuments = this.isDocuments.bind(this);
+		this.isNotifications = this.isNotifications.bind(this);
+	}
+
+	isActive(route) {
+		if (window.location.pathname.endsWith(route)) {
+			return "nav-active";
+		}
+		else {
+			return "";
+		}
+	}
+
+	isHome(){
+		if (window.location.pathname.endsWith("/home")) {
+			return <img src={require('../components/navbar-icons/leaderboard-red.png')}
+						className="leaderboard-icon-grey" alt=''></img>
+		}
+		else {
+			return <img src={require('../components/navbar-icons/leaderboard-grey.png')}
+						className="leaderboard-icon-grey" alt=''></img>
+		}
+	}
+
+	isScoreCapture(){
+		if (window.location.pathname.endsWith("/scorecapture")) {
+			return <img src={require('../components/navbar-icons/add-score-red.png')}
+						className="add-score-icon-grey" alt=''></img>
+		}
+		else {
+			return <img src={require('../components/navbar-icons/add-score-grey.png')}
+						className="add-score-icon-grey" alt=''></img>
+		}
+	}
+
+	isProfile(){
+		if (window.location.pathname.endsWith("/profile")) {
+			return <img src={require('../components/navbar-icons/profile-red.png')}
+						className="profile-icon-grey" alt=''></img>
+		}
+		else {
+			return <img src={require('../components/navbar-icons/profile-grey.png')}
+						className="profile-icon-grey" alt=''></img>
+		}
+	}
+
+	isDocuments(){
+		if (window.location.pathname.endsWith("/documents")) {
+			return <img src={require('../components/navbar-icons/docs-red.png')}
+						className="docs-icon-grey{" alt=''></img>
+		}
+		else {
+			return <img src={require('../components/navbar-icons/docs-grey.png')}
+						className="docs-icon-grey" alt=''></img>
```

Comment:
If a blind person were to use text-to-speech on your website it would go blank as soon as it arrives at the img tag. Fill them with alt descriptions.

Reply:
I sorted it out man